connector_test.go

```diff
@@ -111,6 +111,24 @@ func getConfigStruct(t *testing.T) kubernetes.Config {
 		t.Skipf("Skipping test, load kubeconfig file from user home directory (%v)", err)
 	}
 
+	var certData *string
+	var keyData *string
+	var caData *string
+	if kubeconfig.CertData != nil {
+		certDataStr := string(kubeconfig.CertData)
```

Comment:
Does this need to be done in two steps? It's already a string, so you should be able to just derference it.

Reply:
`kubeconfig.*Data` are `[]byte` that are nil if insecure is set to `true`. `kubernetes.Connection` now have string pointers so I need to unwrap the byte value into a string and then reference it in pointers and then assign them to the connection struct (as far as I know), can you show me another way to do that?